packages/daisyui/src/templates/GridTemplate/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './GridTemplate'; 
```

Comment:
Adding blank line at end to remove the red circled dash
```suggestion
export { default } from './GridTemplate'; 
export * from './GridTemplate';

```